Countries Info/API.swift

```diff
@@ -0,0 +1,473 @@
+// @generated
+//  This file was automatically generated and should not be edited.
+
+import Apollo
+import Foundation
+
+public final class MyCountryQueryQuery: GraphQLQuery {
```

Comment:
It looks like this one should be deleted in favour of `Countries Info/GraphQL/API.swift `